setup.py

```diff
@@ -38,7 +38,7 @@ def find_version(*file_paths):
     zip_safe=True,
     python_requires=">=3.6",
     install_requires=[
-        "cloudformation-cli>=0.1.10,<0.2",
+        "cloudformation-cli>=0.1.11,<0.2",
         # "aws-lambda-builders>=1.0,<2.0",
         "zipfile38>=0.0.2,<0.2",
```

Comment:
we should also set zipfile38 to minimum 0.0.3 in which the license file was added